uint32 n_stacked
geometry_msgs/PoseStamped location_stack
---
PerceptionMSG perception_msg